src/inputs/mod.rs

```diff
@@ -116,100 +117,134 @@ pub(crate) struct InputConfig;
 ///         when the process restarts to pick up where it left
 ///         off. Defaults to false.
 ///
-///     messages_per_epoch: (integer) Defines maximum number of
-///         messages per epoch.  Defaults to `1`. If the consumer
-///         times out waiting, the system will increment to the next
-///         epoch, and fewer (or no) messages may be assigned to the
-///         preceding epoch.
+///     epoch_length: (timedelta)
 ///
 /// Returns:
 ///
 ///     Config object. Pass this as the `input_config` argument to
 ///     your execution entry point.
 #[pyclass(module = "bytewax.inputs", extends = InputConfig)]
-#[pyo3(
-    text_signature = "(brokers, group_id, topics, offset_reset, auto_commit, messages_per_epoch)"
-)]
+#[pyo3(text_signature = "(hosts, group_id, topics, tail, offset_reset, auto_commit, epoch_length)")]
 pub(crate) struct KafkaInputConfig {
     #[pyo3(get)]
-    pub brokers: String,
+    pub hosts: Vec<String>,
```

Comment:
Why the rename? I favored `brokers` since that's what Kafka and `rdkafka` use. NBD just curious.

Reply:
I had copy-pasted the Kafka code I used for recovery that didn't have async, and that happen to use `hosts` but `brokers` seems more specific. I'll change them both to that.